### PR TITLE
Generate valid RSS

### PR DESCRIPTION
**Pull request by irfancharania**
Tested simple output feed with http://feedvalidator.org/

### CODE REVIEW

File: spec/models/agents/data_output_agent_spec.rb [hunk 1]
```diff
@@ -170,7 +171,7 @@
               'title' => 'Evolving yet again with a past date',
               'description' => 'Secret hovertext: Something else',
               'link' => 'http://imgs.xkcd.com/comics/evolving0.png',
-              'guid' => event3.id,
+              'guid' => {"contents" => event3.id, "isPermaLink" => "false"},
```
Comment (cantino) [inline spec/models/agents/data_output_agent_spec.rb]: Is this what we want to output in the JSON case?

File: app/models/agents/data_output_agent.rb [hunk 2]
```diff
@@ -100,7 +100,8 @@ def receive_web_request(params, method, format)
       if interpolated['secrets'].include?(params['secret'])
         items = received_events.order('id desc').limit(events_to_show).map do |event|
           interpolated = interpolate_options(options['template']['item'], event)
-          interpolated['guid'] = event.id
+          interpolated['guid'] = {'_attributes' => {'isPermaLink' => 'false'},
```
Comment (cantino) [inline app/models/agents/data_output_agent.rb]: Makes sense!

File: spec/models/agents/data_output_agent_spec.rb (hunk 1 above)
Comment (irfancharania) [inline spec/models/agents/data_output_agent_spec.rb]: I was debating over that... then left it in for consistency

**Comment by cantino** [comment]
Thanks @irfancharania!

**Pull request merged**